### DISCUSSION TITLE
SEF Flexvind tillæg til spotpris

### DISCUSSION THREAD

**Discussion by Capito1958** [rndfm/elspotpris#130] [Ideas]
Hej
SEF har på flexvind produktet hævet spotpris tillægget til 0,136kr ex.moms pr. kWh.

Kan desværre ikke linke til SEF's prisgrafer, da det kræver login.

**Comment by rndfm** [comment]
Kan du klare et skærmbillede og baglænsberegningen?

**Reply by Capito1958** [reply]
![Screenshot_2023-01-11_105738](https://user-images.githubusercontent.com/122435605/211776362-cfe153cd-5e05-4fce-82fb-3a0310a0ea56.jpg)

Her er prisgrafen for idag 11-01-23.
Det ses f.eks at SEF prisen kl. 01 er 0,17 kr. incl. moms, hvor spotprisen samme tidspunkt er 0,00 kr.
Håber det kan bruges.

**Comment by rndfm** [comment]
Jeg siger mange tak.

Har opdateret satsen. Det skulle gerne slå igennem om et øjeblik.
